src/Uno.Templates/content/unoapp/Directory.Build.props

```diff
@@ -8,6 +8,24 @@
   -->
   <Import Project="solution-config.props" Condition="exists('solution-config.props')" />
 
+  <PropertyGroup>
```

Comment:
Maybe it would be better to move this `PropertyGroup` and `Target` lower, so that the more "common" things like `ImplicitUsings` and `Nullable` are higher to top of the file

Reply:
Yep, we'll move this back into the solution-config.props file